### DISCUSSION TITLE
SixLabors.ImageSharp.Drawing.Processing.Processors.Drawing.FillRegionProcessor Method not found

### DISCUSSION THREAD

**Discussion by vinayaccela** [SixLabors/ImageSharp.Drawing#264] [General] [answered]
Environment - 

- Dotnet core 3.1 
- SixLabors.ImageSharp, Version 2.1.1
- SixLabors.ImageSharp.Drawing, Version 1.0.0-beta13

Error message - System.MissingMethodException: Method not found: 'System.Span`1 SixLabors.ImageSharp.ImageFrame`1.GetPixelRowSpan(Int32)'. at SixLabors.ImageSharp.Drawing.Processing.Processors.Drawing.FillRegionProcessor`1.OnFrameApply(ImageFrame`1 source) at SixLabors.ImageSharp.Processing.Processors.ImageProcessor`1.SixLabors.ImageSharp.Processing.Processors.IImageProcessor.Execute() at SixLabors.ImageSharp.Processing.DefaultImageProcessorContext`1.ApplyProcessor(IImageProcessor processor, Rectangle rectangle)

Error occurs in this line - imageContext.FillPolygon(bgColor, points);

**Comment by antonfirsov** [comment, marked as answer]
> SixLabors.ImageSharp.Drawing, Version 1.0.0-beta13

That seems to be an old version. Looks like you updated the core `SixLabors.ImageSharp` package, but not `SixLabors.ImageSharp.Drawing`.

**Reply by vinayaccela** [reply]
Only version available for SixLabors.ImageSharp.Drawing from nuget is 1.0.0-beta13, no other versions are available.

**Reply by antonfirsov** [reply]
beta15 is the latest version:
https://www.nuget.org/packages/SixLabors.ImageSharp.Drawing/1.0.0-beta15

Note that all versions are prerelease.

**Reply by vinayaccela** [reply]
thanks @antonfirsov issue is resolved now
